addons/l10n_eu_service/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_l10n_eu_service_service_tax_rate,access_l10n_eu_service_service_tax_rate,model_l10n_eu_service_service_tax_rate,,1,1,1,1
```

Comment:
This pure data model should not be modified by anyone, but if it ever is, it should only be by the admin or a financial manager. The default ACL should be `1,0,0,0`, with perhaps an extra line `1,1,1,1` for `account.group_account_manager`